modules/context/context.go

```diff
@@ -181,6 +181,10 @@ func (ctx *Context) RedirectToFirst(location ...string) {
 			continue
 		}
 
+		if len(loc) > 1 && loc[0] == '/' && (loc[1] == '/' || loc[1] == '\\') {
```

Comment:
```suggestion
		// some browsers will treat the `Location: /\evil.com` as `Location: https://evil.com`, so it's not acceptable if the second char is '\\' or '/'
		if len(loc) > 1 && loc[0] == '/' && (loc[1] == '/' || loc[1] == '\\') {
```

LGTM, Could we add some comments?